src/kfs_frontend/kfs_utils.cpp

```diff
@@ -176,4 +177,22 @@ void setBatchSize(KFSTensorOutputProto& proto, int64_t batch) {
 void setStringPrecision(KFSTensorOutputProto& proto) {
     proto.set_datatype("BYTES");
 }
+Status getRawInputContentsBatchSizeAndWidth(const std::string& buffer, size_t& batchSize, size_t& width) {
+    size_t offset = 0;
+    size_t tmpBatchSize = 0;
+    size_t tmpMaxStringLength = 0;
+    while (offset + sizeof(int32_t) <= buffer.size()) {
```

Comment:
```suggestion
    while (offset + sizeof(uint32_t) <= buffer.size()) {
```
Use uint32_ when refering to first 4 bytes of a string related to its size